src/services/verify/index.js

```diff
@@ -0,0 +1,71 @@
+import { getDocumentStoreRecords } from "@govtechsg/dnsprove";
```

Comment:
`getIssuersIdentities` and `issuersIdentitiesAllVerified` are implementation details, they dont need to be exported or tested

Reply:
Actually `issuersIdentitiesAllVerified` is as well an implementation, I would suggest to focus testing on `verifyDocument` only and keep different scenario as they are currently described